LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -294,7 +237,17 @@ int __cdecl wmain(int argc, WCHAR *argv[])
         }
     }
 
-    StartMonitors(configFileName);
+    LoggerSettings settings;
+    //read the config file
+    bool configFileReadSuccess = OpenConfigFile(configFileName, settings);
+
+    //start the monitors
+    if (configFileReadSuccess)
+    {
+        StartMonitors(settings);
+    } else {
+        logWriter.TraceError(L"Invalid configuration file.");

```

Comment:
Also "Invalid json configuration file." can be used to make the message more clear.

Reply:
This is an existing bit. It can be changed in a different PR after the refactor.